etc/behat/services/pages.xml

```diff
@@ -40,6 +40,9 @@
         <parameter key="sylius.behat.page.admin.country.create.class">Sylius\Behat\Page\Admin\Country\CreatePage</parameter>
         <parameter key="sylius.behat.page.admin.country.index.class">Sylius\Behat\Page\Admin\Country\IndexPage</parameter>
         <parameter key="sylius.behat.page.admin.country.update.class">Sylius\Behat\Page\Admin\Country\UpdatePage</parameter>
+        <parameter key="sylius.behat.page.admin.currency.create.class">Sylius\Behat\Page\Admin\Currency\CreatePage</parameter>
+        <parameter key="sylius.behat.page.admin.currency.index.class">Sylius\Behat\Page\Admin\Currency\IndexPage</parameter>
```

Comment:
I think [we do](https://github.com/Sylius/Sylius/pull/4604/files#diff-b2b48f06b5587eca9ce87915f0456693R20) 😄